packages/manager/apps/dedicated/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "dependencies": {
     "@ovh-ux/manager-account-migration": "^0.0.0 || ^1.0.0",
-    "@ovh-ux/manager-account-sidebar": "^2.0.0 || ^3.0.0",
+    "@ovh-ux/manager-account-sidebar": "^2.0.0 || ^3.0.0",
```

Comment:
Important to notice that several `package.json` file has non-breaking space.
It can be removed safely.

See: https://github.com/ovh/manager/pull/5360/files?file-filters%5B%5D=.json